### DISCUSSION TITLE
Manual Install for Ad-block

### DISCUSSION THREAD

**Discussion by ghost** [SpotX-Official/SpotX#257] [Ideas]
Is there a possibility to support manual installation of the ad-blocking functionality?

This would be for those of us in a managed environment where running arbitrary scripts is frowned upon/completely blocked. I have really enjoyed using BlockTheSpot and I saw that those efforts have joined here. Thanks for the consideration 🤗

**Comment by amd64fox** [comment]
At the moment there is no manual install for SpotX, but Bts has it, [try it](https://github.com/mrpond/BlockTheSpot#manual-installation).

**Comment by ghost** [comment]
For sure. I've been using the manual install for bts with great success! Just putting the idea out there in anticipation of the day that Spotify breaks bts since bts is EoL.

**Comment by fernandom241** [comment]
please let there be manual install for spot X, i am actively begging you, bts was amazing but it just stopped working today, like it was insane that i could just copy and paste two files and immediately be free of ads. right now every time i try to download spotx it just count as malware and my Kaspersky just disinfect again and again

**Reply by amd64fox** [reply]
Kaspersky has one of the worst heuristic analysis among AV, too many false positives

just disable it while installing Spotx

**Reply by SirGryphin** [reply]
I've just switch from BTS to SpotX and I used the automatic install using Powershell code and had no problem, I'm also using Kaspersky, not sure if your using .bat file or running the command in PS yourself.

Only thing changed for me using SpotX instead of BTS is Spotify now has a new layout with new feature so mine must of been well out of date 🤣
